definitions/npm/@testing-library/react_v10.x.x/flow_v0.104.x-/react_v10.x.x.js

```diff
@@ -202,13 +202,31 @@ declare module '@testing-library/react' {
     eventProperties?: TInit
   ) => boolean;
 
+  declare type Screen<Queries = GetsAndQueries> = {
+    ...Queries,
```

Comment:
Spreading in an inexact object doesn't behave a intended.

Reply:
What would be the more appropriate approach? I was trying to follow previous discussions and carry over the latest

Reply:
Try to use exact as much as possible, if not possible intersection?